### PR TITLE
Basic app skeleton using Rails

### PR DESCRIPTION
**Pull request by ghost**
App skeleton generated using 'Rails new' command.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use Puma as the app server
+gem 'puma', '~> 3.0'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
+# Use CoffeeScript for .coffee assets and views
+gem 'coffee-rails', '~> 4.2'
```
Comment (gotar) [inline Gemfile]: are you guys agree you will use coffee for js?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
+  gem 'web-console'
+  gem 'listen', '~> 3.0.5'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (gotar) [inline Gemfile]: clean comments

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```
Comment (gotar) [inline README.md]: do not include default README you have empty file on master already exists

**Comment by hav3loc** [comment]
# Board&Roll
readme (temat, opis, kto robi, jak uruchomic, testowac)

Board&Roll is an application used to find board games meetings in your city with goal to allow meetups for people with love to fancy board games.

Application adjuvant to organization of board game meetings. 
## Gems
*
## Credits
* [Kamil Filipowski](https://github.com/KamilFilip)
* [Maciej Michalik](https://github.com/maciek1995)
* [Bernard Pietraga](https://github.com/bpietraga)
* [Alicja Raszkowska](https://github.com/trueskawka)
* [Karolina Rostek](https://github.com/Carolsien)
* [Katarzyna Sikorska](https://github.com/Snieezy)
* [Piotr Sordyl](https://github.com/hav3loc)

## License

File: app/assets/javascripts/cable.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
+  this.App || (this.App = {});
+
+  App.cable = ActionCable.createConsumer();
+
+}).call(this);
```
Comment (gotar) [inline app/assets/javascripts/cable.js]: Do you need ActionCable in your app?

Comment (hav3loc) [inline app/assets/javascripts/cable.js]: Ok, we understand, we'll take a closer look through the files generated.

File: bin/spring [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+unless defined?(Spring)
+  require 'rubygems'
+  require 'bundler'
+
+  if (match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m))
+    Gem.paths = { 'GEM_PATH' => [Bundler.bundle_path.to_s, *Gem.path].uniq.join(Gem.path_separator) }
+    gem 'spring', match[1]
+    require 'spring/binstub'
+  end
+end
```
Comment (gotar) [inline bin/spring]: Not sure if having spring will be good. it leads to many errors (hard to debug) if not really needed I will disable it

File: test/test_helper.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  fixtures :all
+
+  # Add more helper methods to be used by all tests here...
+end
```
Comment (gotar) [inline test/test_helper.rb]: use rspec as a test framework

**Comment by gotar** [comment]
👍

**Comment by hav3loc** [comment]
Everything seems to be properly configured, can we merge? @mentors

**Comment by gotar** [comment]
I already give you +1

**Pull request merged**